src/app/lists/dataview.tsx

```diff
@@ -0,0 +1,82 @@
+"use client";
+
+import Table from "@/components/Tables/Table";
+import supabase from "@/supabase";
+import React from "react";
+import { styled } from "styled-components";
+
+const TableDiv = styled.div`
+    margin: 20px;
+    padding: 20px;
+    background-color: ${(props) => props.theme.surfaceBackgroundColor};
+
+    border: solid 1px ${(props) => props.theme.surfaceForegroundColor};
+    border-radius: 1rem;
+`;
+
+const StyledTable = styled(Table)`
+    width: 100%;
+    background-color: transparent;
+`;
+
+type ListRow = { [headerKey: string]: string };
+
+const ListsDataView = async (): Promise<React.ReactElement> => {
+    // const rawData = (await supabase.from("lists").select("*")).data;
```

Comment:
the brackets here suggest that two 'logical' things are being done - so should be two constants!
(There's no 'cost' on variables, especially since this will likely compile down to the same thing, so please do write nice clean code!)
e.g. `const response = await supabase...select(); const rawData = response.data;`

Reply:
Changed